src/desktop/apps/fairs/query.coffee

```diff
@@ -1,8 +1,8 @@
 module.exports = '''
   query FairsQuery($page: Int) {
-    featured_fairs: ordered_sets(key: "art-fairs:featured") {
+    featured_fairs: orderedSets(key: "art-fairs:featured") {
       fairs: items {
-        ... on FeaturedLinkItem {
+        ... on FeaturedLink {
```

Comment:
Not sure I fully followed the change btw v1 and v2 when it comes to this type, but this _does_ work.

Reply:
Sweet! I think @dzucconi may have done some of the V2 updates for featured links? I don't fully recall these ones.